tools/generate-readme-ts.js

```diff
@@ -1,4 +1,4 @@
-import del from 'del';
+import del from 'del'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
why are we disabling this? isn't `del` in the package.json ?

Reply:
eslint was complaining.